kokoro/gcp_windows/kokoro_build.bat

```diff
@@ -26,10 +26,12 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 
 :: Package build artifacts into a zip for integration in the installer.
 cd %REPO_ROOT%\build\package
-ren bin Orbit
+Xcopy /E /I bin Orbit
+del bin\Orbit.pdb
```

Comment:
del Orbit\Orbit.pdb?

Reply:
And another comment: Irina ist currently adding automatic symbol uploading support (see pull/548). So there will be some more *.pdb files from Qt inside of bin\. Even some in subfolders. So we should coordinate this change with pull/548

Reply:
The *.pdb from Qt should be inside `build\bin` folder. I didn't change the package() function in conanfile, so the content of `build\package\bin` should remains the same.

Reply:
One more thing: I added kokoro_release_build.bat for the release builds.  
Should we duplicate the changes there?

Reply:
> The *.pdb from Qt should be inside `build\bin` folder. I didn't change the package() function in conanfile, so the content of `build\package\bin` should remains the same.

Good point. We should probably also copy them in the package-function. Otherwise we won't have them in the bucket. And we want to have them in the backup in case the automatic upload fails. Otherwise it get's tricky to repeat the process manually. Can you make that change?

Reply:
> One more thing: I added kokoro_release_build.bat for the release builds.
> Should we duplicate the changes there?

Yes. Daniel?

Reply:
Droping this because of git weirdness I produced. 
Replacement is here: https://github.com/google/orbit/pull/559